metron-platform/metron-pcap-backend/src/test/java/org/apache/metron/pcap/integration/PcapTopologyIntegrationTest.java

```diff
@@ -220,13 +208,13 @@ public static void setupTopology(Function<Properties, Void> updatePropertiesCall
     clearOutDirs(inputDir, interimResultDir, outputDir);
 
     File baseDir = new File(new File(targetDir), BASE_DIR);
-    //Assert.assertEquals(0, numFiles(outDir));
-    Assert.assertNotNull(topologiesDir);
-    Assert.assertNotNull(targetDir);
+    //assertEquals(0, numFiles(outDir));
```

Comment:
I know it was previously commented out, but should we keep this assertion?

Reply:
Honestly, I have no idea without digging what the test actually does and if that assertion would be meaningful or not. I can look into it more if you want.

Unrelated, this file is absolutely lousy with `System.out`. We really should get our test output cleaned up and use the utilities we have for managing logging more.